lib/matplotlib/transforms.py

```diff
@@ -1474,7 +1474,7 @@ def transform(self, values):
         Returns
         -------
         array
-            The output values as NumPy array of length :attr:`input_dims` or
+            The output values as NumPy array of length :attr:`output_dims` or
```

Comment:
There are more locations to possibly change this. I cannot get my head around if it should be `output_dims` or `input_dims` though...

Reply:
99.9% sure `input_dims`!

Reply:
I think "output_dims", but it is the case that in many cases they are both equal to 2 so it is correct by chance....

Reply:
Ooops, yes, I agree.  Sorry for being so certain yet so wrong.

Reply:
OK! I've updated the remaining ones. Unfortunately, the subclasses documentation does not inherit the attributes. Not sure if that can be done in a better way than tell that all inherited attributes should be shown?

Anyway, it is now correct (assuming that `output_dims` is indeed correct), although the links don't work.